<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="statusTip">
        <string>Set N.</string>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_struct">
       <property name="statusTip">
        <string>Choose structure.</string>
       </property>
       <item>
        <property name="text">
         <string>stack</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>deque</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="CB_rezults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Choose one of rezults.</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TB_actions">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>111</width>
      <height>209</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Array of actions.</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="B_run_visual">
       <property name="text">
        <string>step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="LW_struct_visual">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>Structure visualization.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_one_rezult"/>
    <addaction name="actionSave_all_rezults"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_one_rezult">
   <property name="text">
    <string>Save one rezult...</string>
   </property>
  </action>
  <action name="actionSave_all_rezults">
   <property name="text">
    <string>Save all rezults...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
